tests/article.test.js

```diff
@@ -141,3 +141,30 @@ describe('create  article', () => {
       });
   });
 });
+
+describe('GET /drafts/:slug', () => {
```

Comment:
@truestbyheart can you quickly add a test that shows that when a user who isn't the author requests for a draft article, it returns an error?